lib/tizen_plugins.dart

```diff
@@ -629,3 +633,35 @@ Future<void> renderTemplateToFile(
   await file.create(recursive: true);
   await file.writeAsString(renderedTemplate);
 }
+
+/// See: [_writeFlutterPluginsList] in flutter_plugins.dart
+Future<void> _writeTizenPluginsExtraInfo(
+  FlutterProject project,
+) async {
+  final File extraInfoFile =
+      project.directory.childFile('.flutter-tizen-plugins-extra-info');
+
+  final List<TizenPlugin> plugins = await findTizenPlugins(project);
+  final List<Map<String, Object>> pluginInfo = <Map<String, Object>>[];
+  for (final TizenPlugin plugin in plugins) {
+    pluginInfo.add(<String, Object>{
+      'name': plugin.name,
+    });
+  }
+
+  final Map<String, Object?> result = <String, Object>{};
+  result['info'] =
+      'This is a generated file; do not edit or check into version control.';
+  result['plugins'] = pluginInfo;
+  result['date_created'] = globals.systemClock.now().toString();
+  result['dart_version'] = globals.flutterVersion.dartSdkVersion;
+  result['flutter_version'] = globals.flutterVersion.frameworkVersion;
+  result['engine_version'] =
+      globals.cache.getStampFor(kTizenEngineStampName) ?? '';
+  result['embedder_version'] =
+      globals.cache.getStampFor(kTizenEmbedderStampName) ?? '';
```

Comment:
This is the result from this code.
```yaml
  "dart_version": "3.6.0",
  "flutter_version": "3.27.1",
  "engine_version": "4d673e9827721f16b4b5f12d139b713feaa7ba3d",
  "embedder_version": "3b5d236f482e78e89b6b766c93b9777aba17788d"
```

I think this result is a bit more specific and each type can be distinguished.
like this. what do you think?
```yaml
  "dart_version": [
    "version" : "3.6.0",
    "hash" : "" # hash
  ],
  "flutter_version": [
    "version" : "3.27.1",
    "hash" : ""
  ],
  "engine_version": [
    "version" : "", # or "3.27.1"
    "hash" : "4d673e9827721f16b4b5f12d139b713feaa7ba3d"
  ],
  "embedder_version": [
    "version" : "",
    "hash" : "3b5d236f482e78e89b6b766c93b9777aba17788d"
  ]
```